packages/docusaurus-theme-classic/src/theme/Navbar/index.js

```diff
@@ -77,7 +77,8 @@ function Navbar() {
         target: '_blank',
       }
     : null;
-  const logoImageUrl = useBaseUrl(logo.src);
+  const logoSrc = logo.darkSrc && isDarkTheme ? logo.darkSrc : logo.src;
```

Comment:
Maybe  is it better to move this expression to the hook call?

```js
const logoImageUrl = useBaseUrl(logo.darkSrc && isDarkTheme ? logo.darkSrc : logo.src);
```

Reply:
Both are acceptable.